### ISSUE TITLE
bug: tls cert authority invalid

### ISSUE THREAD

**Issue by dillfrescott** [swiftwave-org/swiftwave#394]
### Describe the bug

When issuing a cert, every browser on every device says the authority is invalid

### Device Information [optional]

```markdown
- OS: Windows, Android
- Swiftwave version: 1.0.5
- Docker version: Docker version 25.0.4, build 1a576c5
```

### Are you working on this issue?

No

**Comment by tanmoysrt** [comment]
Hi @dillfrescott 
Is this error for dashboard or deployed application ?

**Comment by dillfrescott** [comment]
dashboard. I havent got around to issuing a cert for an application yet.

**Comment by tanmoysrt** [comment]
> dashboard. I havent got around to issuing a cert for an application yet.

By default, TLS is enabled, so if you don't issue a certificate it will show error. 

**There can be two ways to resolve this -**
1. If you are trying out on a server, which has public IP address. Run this command to issue certificate.
```
sudo swiftwave tls generate-certificate
```

2. Disable TLS. Then your dashboard will be served over HTTP. Run below command for that
```
sudo swiftwave tls disable
sudo swiftwave service restart
```

> [!WARNING]  
> _If you disable TLS, some websocket based feature for realime logs will not work in dashboard._

**Comment by dillfrescott** [comment]
Okay thank you. Also it wont let me delete ingress rules. it says deleting for nearly an hour and i even tried restarting the server and my browser

**Comment by tanmoysrt** [comment]
> Okay thank you. Also it wont let me delete ingress rules. it says deleting for nearly an hour and i even tried restarting the server and my browser

Is haproxy running ?
```
sudo swiftwave haproxy status
```

**Comment by dillfrescott** [comment]
Yes

**Comment by dillfrescott** [comment]
![Screenshot 2024-03-17 211239](https://github.com/swiftwave-org/swiftwave/assets/56714680/83a0c122-de79-4788-8a34-81ccbc8ccb50)

Tried it on a new fresh vps with a new ip. its just borked...

**Comment by dillfrescott** [comment]
Wait disregard i think i figured it out. i apologize

**Comment by dillfrescott** [comment]
it was an issue with my configuration.

**Comment by tanmoysrt** [comment]
Thanks for the update !
However in v1 there is some bugs, which lead to this kind of behaviour. In that case you can just restart swiftwave and then again press on delete.

In v2, we are taking care all of this bugs and stability issues, Expected to be released on this month end.

**Comment by dillfrescott** [comment]
Excellent!

**Comment by tanmoysrt** [comment]
Fixed in #413 , will be available in v2.0.0 release
